python/xorq/ibis_yaml/tests/test_relations.py

```diff
@@ -83,3 +91,24 @@ def test_limit(compiler, t):
     assert expression["n"] == 10
     roundtrip_expr = compiler.from_yaml(yaml_dict)
     assert roundtrip_expr.equals(expr)
+
+    def test_flightudxf_validate_schema_fail():
+        import xorq.vendor.ibis as ibis
+        schema = ibis.schema({"x": "int64"})
+        input_expr = ibis.table(schema, name="t")
+
+        class DummyUDXF:
+            schema_in_required = "required_schema"
+            @staticmethod
+            def schema_in_condition(sch):
+                return False
+            @staticmethod
+            def calc_schema_out(sch):
+                return None
+
+        try:
```

Comment:
It is better to use [pytest.raises](https://docs.pytest.org/en/stable/reference/reference.html#pytest-raises)